<launch>
    <arg name="vins_path" default="$(find mcvio)/config/">
    <arg name="config_path" default="$(find mcvio)/config/MCVIO/EuRoC/MCVIO_EuRoC.yaml"/>

    <node name="MCVIO_estimator" type="mcvio_estimator", pkg="mcvio", output="screen">
        <param name="config_file" type="string" value="$(arg config_path)">
        <param name="vins_folder" type="string" value="$(arg vins_path)">
        <param name="vins_imu_frame" type="string" value="vins_imu"/>
        <param name="camera_frame" type="string" value="vins_camera"/>
        <param name="vins_world_frame" type="string" value="vins_world"/>
    </node>

</launch>